engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -584,38 +607,30 @@ private static String getDefaultRootFolder()
    */
   public static Path getApexDFSRootDir(FileSystem fs, Configuration conf)
   {
-    String apexDfsRootDir = conf.get(APEX_APP_DFS_ROOT_DIR);
     boolean useImpersonated = conf.getBoolean(StramUserLogin.DT_APP_PATH_IMPERSONATED, false);
     String userShortName = null;
     if (useImpersonated) {
       try {
         userShortName = UserGroupInformation.getCurrentUser().getShortUserName();
       } catch (IOException ex) {
-        LOG.warn("Error getting current/login user name {}", apexDfsRootDir, ex);
+        LOG.warn("Error getting current/login user name in getApexDFSRootDir", ex);
```

Comment:
Remove the method name from the log message (already provided by logger).

Reply:
fixed ( in both cases)